pkg/tbtc/signing.go

```diff
@@ -213,15 +290,29 @@ func (srl *signingRetryLoop) start(
 }
 
 // qualifiedOperatorsSet returns a set of operators qualified to participate
-// in the given signing attempt.
-func (srl *signingRetryLoop) qualifiedOperatorsSet() (
-	map[chain.Address]bool,
-	error,
-) {
+// in the given signing attempt. The set of qualified operators is taken
+// from the set of active operators who announced readiness through
+// their controlled signing group members.
+func (srl *signingRetryLoop) qualifiedOperatorsSet(
+	readyMembersIndexes []group.MemberIndex,
+) (map[chain.Address]bool, error) {
+	// The retry algorithm expects that we count retries from 0. Since
+	// the first invocation of the algorithm will be for `attemptCounter == 1`
+	// we need to subtract one while determining the number of the given retry.
+	retryCount := srl.attemptCounter - 1
```

Comment:
Nope, DKG has a different logic where the counters are separated. Here it makes sense because if we are on attempt `1` we did `0` retries. If we are on attempt `2` we did `1` retry and so on.